dbms/src/Storages/Transaction/RegionBlockReader.cpp

```diff
@@ -208,6 +208,11 @@ bool RegionBlockReader::readImpl(Block & block, const RegionDataReadInfoList & d
         }
         index++;
     }
+    if (with_check)
```

Comment:
should I remove the boolean variable `with_check` and remove the benchmark test now?

Reply:
I think `with_check` can be removed. And the benchmark test can be kept just for the future use(no need to compare the WithCheck and WithOutCheck version).